02-molecules/menus/Menu.component.js

```diff
@@ -11,7 +11,7 @@ const Menu = ({block = 'menu', element, modifiers = [], level, items}) => {
 
   return (
     <ul className={bem(block, element, mods)}>
-      { items.map(({title, url, below}, id) => (
+      { items && items.map(({title, url, below}, id) => (
```

Comment:
What does this do?

Reply:
@ModulesUnraveled I'm evaluating two expressions. In the past you may have written:
```
if (this == that && soylet == people && spaghettios == delicious) {
  then do something;
}
```
Since this expression uses ANDs, the code will be executed left to right and will exit as soon as it hits a false statement, ignoring the rest of the cases.

So for my work, I'm checking if `items` is defined. As long as it has a value, it will evaluate to true. If the first part is true, then the second part is executed.

You'll see this a lot in react, where someone checks if something exists before rendering a component.
```
{ showSidebar && (
  <div className="sidebar>
    Welcome to Thunderdome
  </div>
}
```